rkemmy/food-first
# Feature get all meals #160812268

**What does this PR do?**
Allows admin to get all meals
**Task Description**
* Check that user is logged in
* Check that logged in user is admin
* Fetch all meals from meals table

**Manual Testing**
Open terminal then run:
```git clone https://github.com/rkemmy/food-first.git```

cd into project folder and create a virtual environment then run
```pip install -r requirements.txt```

Test the following endpoint
```GET   /api/v2/meals```
set KEY = Content-Type , VALUE= application/json
JWT Token should be included
KEY = Authorization , VALUE = Bearer, JWT Token from admin login endpoint
Press send

**Relevant User Stories**
[#160812268](https://www.pivotaltracker.com/story/show/160812268)
